admin_guide/cluster_management.rst

```diff
@@ -464,7 +481,7 @@ To set non-sensitive libpq connection parameters to be used for all node connect
   SET citus.node_conninfo =
     'sslrootcert=/path/to/citus.crt sslmode=verify-full';
 
-There is a whitelist of parameters that the GUC accepts. See the :ref:`node_conninfo <node_conninfo>` reference for details.
+There is a whitelist of parameters that the GUC accepts. See the :ref:`node_conninfo <node_conninfo>` reference for details. By default, nodes require connections to be encrypted.
```

Comment:
@begriffs I think we can expand on this. We require it, but more importantly enforce it by defaulting to 'require' for ssl_mode.